src/reducers/form.js

```diff
@@ -34,16 +34,9 @@ const field = (state = initialFieldState, action) => {
   }
 };
 
-const initialPageState = {
-  show: false,
-};
-
-const page = (state = initialPageState) => ({
-  show: !state.show,
-});
-
+// TODO Hydrate initial form state from query string
 const initialFormState = {
-  pages: {},
```

Comment:
I think that's a great idea on both accounts. I think this is ready to merge unless you want another person's take on the changes.